native/cocos/bindings/jswrapper/v8/ScriptEngine.cpp

```diff
@@ -155,43 +158,43 @@ bool jsbConsoleFormatLog(State &state, const char *prefix, int msgIndex = 0) {
                 msg += " " + args[i].toStringForce();
             }
         }
-
-        SE_LOGD("JS: %s%s", prefix, msg.c_str());
+        cc::Log::logMessage(cc::LogType::KERNEL, level
+            ,"JS: %s%s", prefix, msg.c_str());
     }
 
     return true;
 }
 
 bool jsbConsoleLog(State &s) {
-    jsbConsoleFormatLog(s, "");
+    jsbConsoleFormatLog(s, cc::LogLevel::LEVEL_DEBUG, "");
     oldConsoleLog.toObject()->call(s.args(), s.thisObject());
     return true;
 }
 SE_BIND_FUNC(jsbConsoleLog)
 
 bool jsbConsoleDebug(State &s) {
-    jsbConsoleFormatLog(s, "[DEBUG]: ");
+    jsbConsoleFormatLog(s, cc::LogLevel::LEVEL_DEBUG, "[DEBUG]: ");
     oldConsoleDebug.toObject()->call(s.args(), s.thisObject());
     return true;
 }
 SE_BIND_FUNC(jsbConsoleDebug)
 
 bool jsbConsoleInfo(State &s) {
-    jsbConsoleFormatLog(s, "[INFO]: ");
+    jsbConsoleFormatLog(s, cc::LogLevel::INFO, "[INFO]: ");
```

Comment:
Is the string `"[INFO]: "` needed?

Reply:
I think it can be removed, after all it is redundant